SecurityPkg/Library/TcgEventLogRecordLib/TcgEventLogRecordLib.c

```diff
@@ -106,7 +106,7 @@ MeasureFirmwareBlob (
   {
     if (Description != NULL) {
       AsciiSPrint ((CHAR8 *)FvBlob2.BlobDescription, sizeof (FvBlob2.BlobDescription), "%a", Description);
-    } else {
+    } else if (FvName != NULL) {

```

Comment:
this seems unnecessary.  The conditional on line 104 should guarantee description or name is not null.